src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void register_nullUser_notOk() {
+        User nullUser = new User(null, null, null);
+        assertThrows(RuntimeException.class, () ->
```

Comment:
It is recommended to throw more specific exceptions rather than a generic `RuntimeException`. Use custom or appropriate exceptions like `IllegalArgumentException` or `RegistrationException` to clearly indicate the type of error.